examples/ex11p.cpp

```diff
@@ -95,6 +96,14 @@ int main(int argc, char *argv[])
 #ifdef MFEM_USE_STRUMPACK
    args.AddOption(&sp_solver, "-sp", "--strumpack", "-no-sp",
                   "--no-strumpack", "Use the STRUMPACK Solver.");
+#endif
+#ifdef MFEM_USE_MKL_CPARDISO
+   args.AddOption(&pardiso_solver,
+                  "-pardiso",
+                  "--pardiso",
+                  "-no-pardiso",
+                  "--no-pardiso",
+                  "Use the MKL Cluster Pardiso Solver.");
```

Comment:
Can we use a consistent style?

```suggestion
   args.AddOption(&pardiso_solver, "-pardiso", "--pardiso", "-no-pardiso",
                  "--no-pardiso", "Use the MKL Cluster Pardiso Solver.");
```